src/compas/colors/color.py

```diff
@@ -456,6 +470,28 @@ def from_hex(cls, value):
         b = HEX_DEC[value[4:6]]
         return cls(r / 255.0, g / 255.0, b / 255.0)
 
+    @classmethod
+    def from_html_name(cls, name):
```

Comment:
perhaps this doesn't need to mention "html"?

Reply:
true, done